Project/scripts/deploymentAndMinting.ts

```diff
@@ -3,7 +3,8 @@ import { ethers } from "ethers";
 import "dotenv/config";
```

Comment:
do we still need this, I think @drsmog https://github.com/Encode-Club-Solidity-Bootcamp-Group1/Week2-Tokenized-Votes/pull/2/files
split some task, I think we can use @drsmog version?